Utilities/Templates/Modules/PythonCLI/PythonCLIModuleTemplate.py

```diff
@@ -0,0 +1,36 @@
+import itk
```

Comment:
Additionally, would it be possible to use `SimpleITK`, wrapped ITK is not yet enabled by default. 

Thanks for your help